package.json

```diff
@@ -345,6 +343,7 @@
     "@metamask/phishing-controller": "^12.0.1",
     "@metamask/post-message-stream": "^8.0.0",
     "@metamask/ppom-validator": "0.34.0",
+    "@metamask/preinstalled-example-snap": "^0.1.0",
```

Comment:
Would it be reasonable to break out the addition of the `preinstalled-example-snap` into a separate change from bumping the existing snaps packages?